fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -2607,6 +2580,7 @@ ACTOR Future<Void> TagPartitionedLogSystem::recruitOldLogRouters(TagPartitionedL
 					req.tLogLocalities = tLogLocalities;
 					req.tLogPolicy = tLogPolicy;
 					req.locality = locality;
+					req.recoverAt = self->getEnd();
```

Comment:
Is it correct to set "recoverAt" to "self->getEnd()" here? This is because "self->getEnd()" increments "recoverAt", and so the receiver on doing "self->getEnd()" would increment "recoverAt" again?

Reply:
Even if the above logic works correctly I think it is better to store the actual recovery version in "recoverAt" and do the increment logic in "self->getEnd()".